### PR TITLE
Improve HTML semantics

### PR DESCRIPTION
**Pull request by meodai**
- improves the use of HTML5 tags and accessibility.
- some CSS needed to be edited, since some of the rules used an element selector

### CODE REVIEW

**Review by nmattia — APPROVED** [review]
Nice, thanks!

**Comment by meodai** [comment]
I've shrunken some of the labels. But I think its still works nicely. I change them a bit again on the CSS rework

File: src/frontend/src/flows/manage/index.ts [hunk 1]
```diff
@@ -48,19 +49,19 @@ const style = () => html`<style>
     text-align: right;
   }
 
-  .labelWithAction label {
+  .labelWithAction .labelWithAction {
```
Comment (wackyleo459) [inline src/frontend/src/flows/manage/index.ts:52]: Was this intended? Wasn't sure if this was typo.

Comment (wackyleo459) [reply src/frontend/src/flows/manage/index.ts:52]: @meodai

**Pull request merged**